apollo-integration/src/test/kotlin/com/apollographql/apollo/NormalizedCacheTestCase.kt

```diff
@@ -353,7 +356,7 @@ class NormalizedCacheTestCase {
         Predicate<Response<HeroAndFriendsNamesWithIDsQuery.Data>> { response -> !response.hasErrors() }
     )
     val heroWithFriendsFragment = apolloClient.apolloStore.read(
-        HeroWithFriendsFragment.Mapper(),
+        HeroWithFriendsFragmentImpl_ResponseAdapter,
```

Comment:
I like that using the imperative store API forces to reference the fragment implementation.